fledging.qmd

```diff
@@ -6,7 +6,81 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
 and want to continue their adventures in cloud computing, have the knowledge
 and resources to develop their own solutions.
 
-Here is a [presentation on fledging](https://docs.google.com/presentation/d/1-TiPN8bfY6iDL5EVcEuCQSjqFMs3jPSX2MlNWA6no2E/edit#slide=id.g2ece79fddf8_3_73) given at the Earth Science Information Partners
-(ESIP) conference in July, 2024.
+When thinking about cloud computing, we consider two components: where is our 
+data? And where is our compute? In general, if our data is already on our local
+computer, we will work with that data on our computer, so the scope we are 
+considering is how to work with *data that is hosted in the cloud*.
 
-More resources coming soon...
+![](images/data-compute-cloud-grid.png){width="400" fig-align="center"}
+
+The proliferation of data centres to support cloud computing and data storage 
+has a [significant environmental cost](https://www.goclimate.com/knowledge/servers). 
+While most scientific workflows are small in comparison to other cloud uses 
+(e.g., AI and cryptocurrency mining), it is worth taking this into 
+consideration when thinking about "when to cloud".
+
+### Skills learned for working with **cloud-hosted data**
+
+The JupyterHubs provide a shared computing environment for mentors to learn and teach
+workflows for accessing and analyzing cloud-based data. Some of the skills we learn include:
+
+- Conceptual understanding of data in the cloud, and cloud-native data sources
+- Finding and accessing data in Earthdata cloud with earthaccess (Python) and 
+  earthdatalogin (R)
+- Streaming and subsetting cloud data with xarray (Python) and terra/rstac (R)
+- Authenticating with NASA Earthdata Login
+- Ability to ask questions, teach, and develop learning resources
+
+These skills enable efficient use of cloud-hosted data using either *local* or 
+*cloud* compute resources.
+
+### What are fledging paths for **compute**?
+
+Depending on computing needs and data size, there are many options for fledging.
+These may include:
+
+- Continue compute in the cloud using a shared environment (example: JupyterHub)
+    - If your institution offers a JupyterHub instance, this is a great option 
+      to seamlessly transfer your learnings from the Openscapes Hub to a Hub
+      for "real science".
```

Comment:
At some point we probably need to raise the issue of dependency management... e.g. I see students work in a hosted jupyter env, and might assume all jupyter envs have the same set of packages available ...